### PR TITLE
Improve handling of failure to get rates, offline, and restrict usage to main net

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label

- Improve handling of failure to get rates, offline, and restrict usage to main net

### CODE REVIEW

File: src/partners/bity/bity-calls.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ const getExitRates = async () => {
       utils.buildPayload(bityMethods.getFiatRates)
     );
     if (results.error) {
+      // return results.error.message;
```
Comment (kvhnuke) [inline src/partners/bity/bity-calls.js]: comments

File: src/partners/kyber/config/config.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const KyberCurrencies = {
 
 const walletDepositeAddress = '0x4247951c2eb6d0bA38d233fe7d542c8c80c9d46A'; // i.e. where payments go
 const kyberNetworkENS = 'kybernetwork.eth';
-const kyberValidNetworks = ['ETH', 'ROP', 'KOV'];
+const kyberValidNetworks = ['ETH']; //['ETH', 'ROP', 'KOV'];
```
Comment (kvhnuke) [inline src/partners/kyber/config/config.js]: commented code

File: src/translations/interface/en_US.json [hunk 3]
```diff
@@ -115,5 +115,7 @@
   "emailPopOver": "Only used if any problems arise with the deposit, otherwise if omitted the value will be refunded to the sending address if any problems arise.",
   "dashboard": "Dashboard",
   "dashboardSwap": "MEW has partnered with Bity, Kyber Network, Changelly, and Simplex to allow users to swap fiat to crypto, ETH and BTC, ETH and ERC-20.",
-  "availableFor": "Available for iOS<br />and Android"
+  "availableFor": "Available for iOS<br />and Android",
```
Comment (gesseekur) [inline src/translations/interface/en_US.json]: kebab case please!

File: src/layouts/InterfaceLayout/containers/SwapContainer/components/SwapWidget/SwapWidget.vue [hunk 4]
```diff
@@ -928,6 +928,26 @@ export default {
...
                   }
                 };
+              } else if (entry.provider === this.providerNames.changelly) {
+                Toast.responseHandler(
+                  `Failed to retrieve Changelly rate from ${fromCurrency} to ${toCurrency}`,
+                  3
+                );
+              } else if (entry.provider === this.providerNames.bity) {
+                Toast.responseHandler(
+                  `Failed to retrieve Bity rate from ${fromCurrency} to ${toCurrency}`,
```
Comment (gesseekur) [inline src/layouts/InterfaceLayout/containers/SwapContainer/components/SwapWidget/SwapWidget.vue]: Translations if possible!

**Pull request merged**